virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -119,6 +119,9 @@ Some features in this release are currently in Technology Preview. These experim
 
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
+//CNV-52857 and CNV-56324
+* You can now xref:../../virt/vm_networking/virt-setting-interface-link-state.adoc#virt-setting-interface-link-state[manage the link state] of a primary or secondary virtual machine (VM) interface by using the {product-title} web console or the CLI.
```

Comment:
nit: I think rephrasing it would make it clearer - 
`... of a primary or secondary interface of a virtual machine (VM)  by using the ...`
If you don't agree you can leave it as it is and resolve this comment.

Reply:
Thanks @Anatw I would like to keep it as is